hosts.yaml

```diff
@@ -111,7 +111,7 @@ all:
             calculate_rack: true
       vars:
         ansible_user: root
-        location: Row 3 Pod B
+        location: Row3/PodB
```

Comment:
We talked yesterday about lower case names for everything to avoid case mismatch problems. Was that in this context, or something else?

Reply:
It was in this context, but when I wrote it as `row3/poda` `row3/pobd` it just looked less readable. that's the only reason I kept the capitalization to keep it readable.

Reply:
Nah, either way is fine, as long as we're consistent :)